src/apps/deskflow-gui/MainWindow.cpp

```diff
@@ -228,6 +223,18 @@ void MainWindow::setupControls()
     m_appConfig.setLastVersion(kVersion);
   }
 
+  // Hide the security label
+  ui->lblConnectionSecurityStatus->setVisible(false);
+
+  // Setup the log toggle, set its initial state to closed
+  ui->btnToggleLog->setStyleSheet(QStringLiteral("background:rgba(0,0,0,0);"));
+  ui->btnToggleLog->setArrowType(Qt::RightArrow);
+  ui->textLog->setVisible(false);
+  if (m_appConfig.logExpanded()) {
+    ui->btnToggleLog->click();
+    toggleLogVisible(true);
+  }
```

Comment:
This seems to fix it for me.

```suggestion
  toggleLogVisible(m_appConfig.logExpanded());
  if (m_appConfig.logExpanded()) {
    ui->btnToggleLog->click();
  }
```